.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+**What did you expect to happen:**
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+**Environment:**
+
+- Sonobuoy version: 
+- Kubernetes version: (use `kubectl version`):
+- Kubernetes installer & version:
+- Cloud provider or hardware configuration:
+- OS (e.g. from `/etc/os-release`): 
+
+Sonobuoy tarball which contains (*)
```

Comment:
is the indication here that they will provide the tarball from their run?

Reply:
+1 I'm confused by this too

Reply:
Yep.

Reply:
Let me reword this one.